src/components/ble/NimbleController.cpp

```diff
@@ -158,9 +158,11 @@ void NimbleController::StartAdvertising() {
     adv_params.itvl_max = 1651;
   }
 
+  ble_uuid128_t serviceUUIDs[] = {dfuServiceUuid, heartRateServiceUuid};
+
   fields.flags = BLE_HS_ADV_F_DISC_GEN | BLE_HS_ADV_F_BREDR_UNSUP;
-  fields.uuids128 = &dfuServiceUuid;
-  fields.num_uuids128 = 1;
+  fields.uuids128 = &serviceUUIDs;
```

Comment:
I don't think you need the "&", now that it's really a array.

Reply:
Thx :)